src/components/Analisis.jsx

```diff
@@ -2,7 +2,7 @@ import styles from "./Analisis.module.css";
 import { Link as ScrollLink } from "react-scroll";
 import PropTypes from "prop-types";
 
-export default function Analisis({ min, max, total, tipos, componentes }) {
+export default function Analisis({ min, max, total, componentes }) {
```

Comment:
shouldn't tipos be removed from analisisData too ?

Reply:
Ah, yes. Probably gonna find a lot more like these with Typescript. I'll push a new commit for this.